dev-dotnet/dotnet-sdk/files/dotnet-sdk-5.0.203-runtime-locate-ranlib.patch

```diff
@@ -0,0 +1,21 @@
+Pick up the CLR_RANLIB variable properly, even when using Clang as a compiler
```

Comment:
Any of these upstreamed? If so, link

Reply:
No, not sure if upstream would like them, will consider.